### PR TITLE
Create user-story.md

### PR DESCRIPTION
**Pull request by ManApart**
Create a template that fits the team norm.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+#### User Story
+As a issue creator
+
+I want to be able to create issues
+
+So that I can see my issue resolved
+
+#### Acceptance Criteria: 
+- [ ] The issue exists in the list of issues
```
Comment (jakeprem) [inline .github/ISSUE_TEMPLATE/user-story.md]: I think the "meta" examples here and on the User story are pretty confusing

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] Issue has all fields properly filled out
+
+#### Tech Notes: 
+- Notes that are technical in nature
```
Comment (jakeprem) [inline .github/ISSUE_TEMPLATE/user-story.md]: I think this section and the Amigos section are much clearer and more helpful

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- Not necessarily an acceptance criteria, but a hint for the people that play the card
+
+#### Amigos:
```
Comment (jakeprem) [inline .github/ISSUE_TEMPLATE/user-story.md]: Some explanation of what an Amigo is might be helpful as part of this

Comment (ManApart) [reply .github/ISSUE_TEMPLATE/user-story.md]: This is for our team to use, and I think we understand each of these sections. What would you propose?

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 1 above)
Comment (ManApart) [reply .github/ISSUE_TEMPLATE/user-story.md]: These are just here to give us a formatting jump start and have been approved by Carrie. What would you propose as an alternative?

Comment (jakeprem) [reply .github/ISSUE_TEMPLATE/user-story.md]: - [ ] This condition that must be met for the card to be complete
- [ ] E.g. Reaper can do a thing

Exact wording can be whatever but I think having explanations or examples makes it clearer what part of the template is part of the format and what part should be replaced with details specific to the card.

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 3 above)
Comment (jakeprem) [reply .github/ISSUE_TEMPLATE/user-story.md]: + A list of tagged users that should be involved in reviewing (or exiting) this work

Our team will be creating most of the issues on this repo (for now), however this template will be an option presented on our main open source repo. Anyone adding an issue here will see this as an option so we shouldn't assume that they have an in depth knowledge of terms our team uses.

Additionally, making this clear will help provide documentation for new contributors and developers joining the team.

Comment (ManApart) [reply .github/ISSUE_TEMPLATE/user-story.md]: I've changed amigos in the not to be 'users that should be involved in reviewing (or exiting) this work'

**Pull request merged**